numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -41,7 +41,7 @@ def add_newdoc(place, name, doc):
 
 add_newdoc('numpy.core.umath', 'absolute',
     """
-    Calculate the absolute value element-wise.
+    Calculate the absolute value element-wise. Equivalent to ``numpy.abs``.
```

Comment:
I think `np.abs` is better than `numpy.abs`.

I'd put this in another paragraph, since the first line should be a simple summary. Something like _"`np.abs` is a shorthand for this function."_

Reply:
I've made that change, but feel free to `git push -f` over if if you can think of something better.

I'll squash the commits when I merge